py_nextbus/client.py

```diff
@@ -102,15 +102,19 @@ def predictions_for_stop(
         if direction_id:
```

Comment:
Should probably raise an error if there is a direction but no route, as that would seem invalid. Especially since we won’t filter results.